.env.sample

```diff
@@ -168,3 +168,6 @@ SMTP_SECURE=true
 # The default interface language. See translate.getoutline.com for a list of
 # available language codes and their rough percentage translated.
 DEFAULT_LANGUAGE=en_US
+
+# Optionally enable rate limiter at application web server
+ENABLE_RATE_LIMITER=false
```

Comment:
Good default for now, although I'll probably make it default to enabled in the future